src/main.rs

```diff
@@ -179,6 +185,7 @@ fn main() {
     let service_name = matches.value_of("service_name").unwrap_or("");
     let service_number = matches.value_of("service_number").unwrap_or("");
     let service_job_number = matches.value_of("service_job_number").unwrap_or("");
+    let vcs_branch = matches.value_of("vcs_branch").unwrap_or("master");
```

Comment:
Nit: you can define the default value using [default_value](https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.default_value).

Reply:
Much better, thanks for the feedback.